spec/lib/local-links-manager/import/links_importer_spec.rb

```diff
@@ -104,6 +104,42 @@
 
         expect(link.reload.url).to eq 'http://www.example.com/this-is-now-different'
       end
+
+      context 'when a link is missing from the CSV' do
+        it 'is removed from the database' do
+          service = FactoryGirl.create(:service, lgsl_code: 123, label: 'Service 123')
+          interaction1 = FactoryGirl.create(:interaction, lgil_code: 1, label: 'Interaction 1')
+          service_interaction1 = FactoryGirl.create(:service_interaction, service: service, interaction: interaction1)
+
+          interaction2 = FactoryGirl.create(:interaction, lgil_code: 2, label: 'Interaction 2')
+          service_interaction2 = FactoryGirl.create(:service_interaction, service: service, interaction: interaction2)
+
+          local_authority = FactoryGirl.create(:local_authority, snac: '00AB', gss: '123')
+          FactoryGirl.create(:link, local_authority: local_authority, service_interaction: service_interaction1, url: 'http://example.com/first-link')
+          FactoryGirl.create(:link, local_authority: local_authority, service_interaction: service_interaction2, url: 'http://example.com/second-link')
+
+          csv_rows_without_second_link = [
+            {
+              lgil_code: '1',
+              lgsl_code: '123',
+              snac: '00AB',
+              url: 'http://www.example.com/first-link',
+            },
+          ]
+          stub_csv_rows(csv_rows_without_second_link)
+
+          ENV['MIN_LINK_COUNT'] = '1'
```

Comment:
Would it be worth adding another test here to document the behaviour when the `MIN_LINK_COUNT` is not met?